### ISSUE TITLE
Data Visualizations

### ISSUE THREAD

**Issue by cyouh95** [deucebrixon/Codecademy_data#1]
Great job with the `matplotlib` visuals, including remembering to label/title all your figures! And alternatively, instead of putting all your figures in the end of your slides as part of the appendix, you could also embed them into the relevant slides, as that may be easier to refer to in the presentation.

**Comment by deucebrixon** [comment]
Yeah, I thought about that, as well.  I think I was thrown off a bit by the instructions.
